fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -532,6 +532,15 @@ private boolean processModifyColumn(ModifyColumnClause alterClause, OlapTable ol
 
         Column oriColumn = schemaForFinding.get(modColIndex);
 
+        for (Index index : olapTable.getIndexes()) {
+            if (index.getIndexType() == IndexDef.IndexType.GIN) {
+                if (index.getColumns().contains(oriColumn.getName()) &&
```

Comment:
is index column name case sensitive?

Reply:
I tested this case, it seems that this code has consider the case sensitive problem.